packages/push/src/instruction/printing/char.rs

```diff
@@ -0,0 +1,150 @@
+use std::io::Write;
+
+use super::super::{Instruction, instruction_error::PushInstructionError};
+use crate::{error::InstructionResult, instruction::NumOpens, push_vm::push_io::HasStdout};
+
+/// An instruction that "prints" a single character.
```

Comment:
maybe `single, predetermined character` to avoid confusion with printing a character from the char stack?